common/src/main/java/org/opensearch/ml/common/transport/upload_chunk/MLUploadModelChunkRequest.java

```diff
@@ -29,22 +29,22 @@
 @ToString
 public class MLUploadModelChunkRequest extends ActionRequest {
 
-    MLUploadModelChunkInput mlUploadInput;
+    MLUploadModelChunkInput uploadModelChunkInput;
```

Comment:
If we are using register for full model model, why not use register model chunk here?

Reply:
I'd suggest go ahead and review other parts. We can create a new PR for the model chunk APIs if needed.